examples/gno.land/r/demo/realm_banker/gno.mod

```diff
@@ -0,0 +1 @@
+module gno.land/r/demo/realm_banker
```

Comment:
<img width="514" alt="Screenshot 2023-12-07 at 8 32 17 PM" src="https://github.com/gnolang/gno/assets/37576387/92d1a580-2a53-4a74-9057-cd751b876620">

maybe now its time to run gno mod tidy

Reply:
I'm keep forgetting about tidy T.T

1b81cb4c42276a3fc0b932d717daca5baac079d0